poc/vdaf_pine.py

```diff
@@ -841,3 +842,38 @@ def __init__(self,
             num_wr_successes=num_wr_successes,
             vdaf_id=self.ID
         )
+
+
+class Pine40HmacSha256Aes128(Pine[Field40, XofHmacSha256Aes128]):
+    test_vec_name = "Pine40HmacSha256Aes128"
+    ID = 0xFFFF1006
+
+    def __init__(
+        self,
+        l2_norm_bound: int,
+        num_frac_bits: int,
+        dimension: int,
+        chunk_length: int,
+        chunk_length_norm_equality: int,
+        num_shares: int,
+        num_proofs: int = 4,
+        num_proofs_norm_equality: int = 1,
```

Comment:
How did you choose these?

Reply:
`num_proofs` is chosen from running the script [poc/find_wr_params.sage](https://github.com/junyechen1996/draft-chen-cfrg-vdaf-pine/pull/105/files/b7860b3c817176b92fd33c00debc76b21697ff2a#diff-df465b6a42e5628b0ad4fce73aa2e6fa5c635509f89a12dc07dc8d2857d98731)